Snakefile

```diff
@@ -81,54 +86,48 @@ def subtypes_by_subtype_wildcard(wildcards):
         "`subtypes_by_subtype_wildcard` -- is there a typo in the subtype you are targetting?")
     return(db[wildcards.subtype])
 
-if LOCAL_INGEST:
-    rule copy_sequences_from_ingest:
-        output:
-            sequences = "data/{segment}/sequences.fasta",
-        params:
-            sequences = lambda w: f"ingest/{INGEST_SOURCE}/results/sequences_{w.segment}.fasta"
-        shell:
-            """
-            cp {params.sequences} {output.sequences}
-            """
-
-    rule copy_metadata_from_ingest:
-        output:
-            metadata = "data/metadata.tsv",
-        params:
-            metadata = f"ingest/{INGEST_SOURCE}/results/metadata.tsv",
-        shell:
-            """
-            cp {params.metadata} {output.metadata}
-            """
-
-else:
-    rule download_sequences:
-        output:
-            sequences = "data/{segment}/sequences.fasta",
-        params:
-            s3_src=S3_SRC,
-        shell:
-            """
-            aws s3 cp {params.s3_src:q}/{wildcards.segment}/sequences.fasta.zst - | zstd -d > {output.sequences}
-            """
-
-    rule download_metadata:
-        output:
-            metadata = "data/metadata.tsv",
-        params:
-            s3_src=S3_SRC,
-        shell:
-            """
-            aws s3 cp {params.s3_src:q}/metadata.tsv.zst - | zstd -d > {output.metadata}
-            """
+rule download_sequences:
+    output:
+        sequences = f"data/{S3_SRC.get('name', None)}/sequences_{{segment}}.fasta",
+    params:
+        address=lambda w: S3_SRC.get('sequences', None).format(segment=w.segment)

```

Comment:
Yeah, however you can't use `S3_SRC['sequences']` because this is expanded when Snakemake initialises regardless of what path the workflow will take. I.e. if you are using local ingest, and thus `S3_SRC={}`, you will get a `KeyError`. I thing explicitly checking for keys (as per [this comment](https://github.com/nextstrain/avian-flu/pull/72#discussion_r1680043743)) will guard against this.

Reply:
Resolving this, as we now include an explicit check for these keys